service/src/main/java/org/cbioportal/genome_nexus/service/mixin/IntergenicConsequencesMixin.java

```diff
@@ -0,0 +1,19 @@
+package org.cbioportal.genome_nexus.service.mixin;
+
+import com.fasterxml.jackson.annotation.*;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class IntergenicConsequencesMixin
+{
+    @JsonProperty(value="impact", required = true)
+    private String impact;
+
+    @JsonProperty(value="variantAllele", required = true)
```

Comment:
check if this works with an empty database. This mixin should map to the original JSON response from VEP

Reply:
We just tested this together :) - it only works after being cached once (because the whole response is stored inside the database and the database model does map correctly)

Filed an issue, we should prolly get rid of service mixins completely: https://github.com/genome-nexus/genome-nexus/issues/268